lib/src/rules/avoid_setters_without_getters.dart

```diff
@@ -63,6 +63,7 @@ class AvoidSettersWithoutGetters extends LintRule {
       NodeLintRegistry registry, LinterContext context) {
     var visitor = _Visitor(this);
     registry.addClassDeclaration(this, visitor);
+    registry.addEnumDeclaration(this, visitor);
```

Comment:
Should we also be looking at mixin declarations?

Reply:
Maybe?  I added a todo since this could introduce new diagnostics and we'd want to do a little vetting of impact first.